packages/netsuite-adapter/src/config.ts

```diff
@@ -160,6 +164,12 @@ const clientConfigType = createMatchingObjectType<SdfClientConfig>({
     installedSuiteApps: {
       refType: new ListType(BuiltinTypes.STRING),
     },
+    maxFileCabinetSize: {
```

Comment:
As said before, I think we should rename it so it would be more self explained
```suggestion
    maxFileCabinetSizeInGB: {
```